### ISSUE TITLE
Traffic signs icons

### ISSUE THREAD

**Issue by wackyweasel** [oshadajay/CeyRo#2]
Hi, 

In the repository, there is the image "classes_grid.png" with all traffic sign icons. I would need to have them in higher resolution, with transparent background if you have them. Is it possible to add them to the repository, or find a way for you to send them to me?

It would be greatly appreciated, 

Thanks

**Comment by oshadajay** [comment]
Hi, I think the image is inside the folder `figures`. Would that satisfy your requirements?

**Comment by wackyweasel** [comment]
No, I need higher resolution and transparent backgrounds. In case you had them. But that's fine, we found most of them online and made the rest with inkscape.
Thanks!
